docs/offboarding.rst

```diff
@@ -0,0 +1,250 @@
+- If the user is still holding on to any *Secure Viewing Station* USB drive or
+  backup
+- If you have any other reason to believe the *Submission Private Key* or the
+  entire *Secure Viewing Station* USB may have been copied or compromised.
+
+You should still keep the old key on the *Secure Viewing Station*, or else you
+will not be able to decrypt submissions that were sent to you while that key
+was in effect.
+
+**You will need:**
+
+- The *Admin Workstation*
+- The *Secure Viewing Station*
+- A *Transfer Device* (LUKS-encrypted USB drive)
+
+On the Secure Viewing Station
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Boot into the *SVS*, and open the **Passwords and Keys** application. We will
+change the name of the current SecureDrop submission key to avoid mixing
+up the old and new keys.
+
+Click on **GnuPG Keys** on the lefthand side of the pane, and select the
+SecureDrop Application Key from the list of available keys.
+
+|select securedrop key|
+
+Double-click the key, and in the **Names and Signature** tab, add a name such
+as "OLD <Your Organization> SecureDrop Submission Key - Do Not Delete - Retired
+<Date>". (This is a local-only change to stop you from mixing up the old and
+new keys).
+
+|edit key name|
+
+Once you have done that, you can delete the original name for the key (but not
+the key itself!), so that the only name you see is "OLD <Your Organization>
+Submission Key - Do Not Delete - Retired <Date>".
+
+|delete key name|
+
+Now :doc:`follow the instructions <generate_submission_key>` to create a PGP
+key on the *Secure Viewing Station*. This will be your new *Submission Key.*
+Copy the fingerprint and new *Submission Public Key* to your *Transfer Device*.
+
+.. |select securedrop key| image:: images/offboard/passwords_keys_sd_key.png
+.. |edit key name| image:: images/offboard/add_key_name.png
+.. |delete key name| image:: images/offboard/delete_key_name.png
+
+On the Admin Workstation
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Take the *Transfer Device* with the new *Submission Public Key* and fingerprint
+to your *Admin Workstation*. As you did during the initial install, copy the
+public key, ``SecureDrop.asc``, to the ``install_files/ansible_base/``
+directory, replacing the existing public key file that is there.
+
+Then, from the ``~/Persistent/securedrop`` directory, run
+
+    .. code:: sh
+
+      ./securedrop-admin sdconfig
+
+If the new public key that you placed in ``install_files/ansible_base`` has the
+same name the same as the old public key, ``SecureDrop.asc``, the only part of
+the configuration you will change is the SecureDrop *Submission Key*
+fingerprint, which you will update with the fingerprint of your new key.
+
+Once you have completed the above, run
+
+    .. code:: sh
+
+      ./securedrop-admin install
+
+to push the changes to the server.
+
+You may want to immediately create a test submission, then use a Journalist
+account to log into the *Journalist Workstation* and download it to a *Transfer
```

Comment:
Journalist Interface, not Journalist Workstation. Also It might be better to just ask folks to download and decrypt the submission rather than documenting how here.